### DISCUSSION TITLE
Favorites Disappear

### DISCUSSION THREAD

**Discussion by pro71** [community/community#110762] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Problem: In version 10.0.19042 of the HPC-HC x64 player, windows 10, 20H2: 
From week to week my favorite videos all disappear, or only a few, or most of them disappear. I have tried uninstalling and then downloading the program, but it doesn't help. I believe the same problem may have existed in my previous version, but do not recall for certain. 
What does anyone suggest? 
(I am not a skilled tech guy, or understand much of the jargon used in the "options" of the HPC-HC)

Thank you for any help.
- pro71

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @pro71 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on HPC-HC x64 players. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
